plugins/woocommerce/src/Admin/Notes/DeprecatedNotes.php

```diff
@@ -17,7 +17,7 @@
  *
  * @deprecated since 1.7.0, use Note
  */
-class WC_Admin_Note extends DeprecatedClassFacade {
+class WC_Admin_Note extends DeprecatedClassFacade { // phpcs:ignore
```

Comment:
Rather than adding all of the `phpcs:ignore` comments to the code, exclusion rules could be added to the phpcs config.

There's a few in place already:

https://github.com/woocommerce/woocommerce/blob/31bcf06ac8cbd2041c89cde3a965798a1200fd23/phpcs.xml#L102-L130

Reply:
Thank you! That's a good idea. Updated in https://github.com/woocommerce/woocommerce/pull/32591/commits/486cd2e3abc3336e80acc5cc5c99acb6c56e729b